sdk/servicebus/service-bus/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## 7.0.0-preview.5 (Unreleased)
 
+### Breaking Changes
+
+- Added Async iterable iterators with pagination support for all the listing methods like `getQueues()`, `getTopics()`,`getQueuesRuntimeInfo()`, etc.
+  [PR 9951](https://github.com/Azure/azure-sdk-for-js/pull/9951)
+  - Please refer to the examples in the `samples` folder - [listingEntities](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/servicebus/service-bus/samples/typescript/src/advanced/listingEntities.ts)
```

Comment:
```suggestion:
### Breaking Changes

- Added Async iterable iterators with pagination support for all the listing methods like `getQueues()`, `getTopics()`,`getQueuesRuntimeInfo()`, etc.
  [PR 9951](https://github.com/Azure/azure-sdk-for-js/pull/9951)
- Please refer to the examples in the `samples` folder - [listingEntities](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/servicebus/service-bus/samples/typescript/src/advanced/listingEntities.ts)
```

Reply:
the link for the file should be "blob" instead of "tree"

Reply:
Updated